tutorial/aqua/quick_start.md

```diff
@@ -0,0 +1,186 @@
+
+For the purpose of this section, we will use a greeting service, which
+
+* has only one exposed function that allows us to to either Hi or Bye an input parameter _name_: "Hi {}" or "Bye {}"
+* is hosted on a node with peer id  12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA
+* has a unique identifier, i.e. service id, c9a315de-4fe2-4730-8f40-9209428383bc
+
+We can check the availability of the service module using the `fldist` tool. In your VSCode terminal run:
+
+```bash
+fldist --node-id 12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA  get_modules --pretty| grep -B 13 -A 1  greeting
+  {
+    "config": {
+      "logger_enabled": true,
+      "logging_mask": null,
+      "mem_pages_count": 100,
+      "mounted_binaries": null,
+      "wasi": {
+        "envs": null,
+        "mapped_dirs": null,
+        "preopened_files": []
+      }
+    },
+    "hash": "9af450928626beefab6c055bd6d6ddb4ae9c59344614cee7083e7be7d5fc93c2",
+    "name": "greeting"
+  },
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+Now that we know our service exists, we can check on its interface(s) also with the`flidst` commandline tool:
+
+```bash
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial# fldist --node-id 12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA  get_interface --id c9a315de-4fe2-4730-8f40-9209428383bc --expand
+{
+  "function_signatures": [
+    {
+      "arguments": [
+        [
+          "name",
+          "String"
+        ],
+        [
+          "greeter",
+          "Boolean"
+        ]
+      ],
+      "name": "greeting",
+      "output_types": [
+        "String"
+      ]
+    }
+  ],
+  "record_types": []
+}
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+That is, our service takes a _String_ parameter and a _boolean_ parameter to either return "Hi %name%" or "Bye %name%". With that, we have all the information needed to use our distributed service. 
+
+Create a new directory *aqua-tutorial* in your workspace and add am *aqua* dir so that you have a structure like about so:
+
+```bash
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial# ls -lrth
+total 4.0K
+drwxr-xr-x 2 root root 4.0K Jun  1 23:59 aqua
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+In the *aqua* dir, add the following Aqua script *greeter.aqua*:
+
+```aqua
+-- greeter.aqua                             <-- 1
+service Local("returnService"):             <-- 2
+  run: string -> ()
+
+service Greeting("service-id"):             <-- 3
+    greeting: string, bool -> string
+
+func greeter(name: string, greet: bool, node: string, service_id: string):  <-- 4
+    on node:                                                      
+      Greeting service_id
+      res <- Greeting.greeting(name, greet)
+    Local.run(res)
```

Comment:
Let's change `<--` to `--<` just so it compiles?
```suggestion
-- greeter.aqua                             --< 1
service Local("returnService"):             --< 2
  run: string -> ()

service Greeting("service-id"):             --< 3
    greeting: string, bool -> string

func greeter(name: string, greet: bool, node: string, service_id: string):  --< 4
    on node:                                                      
      Greeting service_id
      res <- Greeting.greeting(name, greet)
    Local.run(res)
```

Reply:
btw, I've improved fldist a little, so now you can get rid of `Local("returnService")` if you want, and go with a shorter equiv version:
```scala
service Greeting("service-id"):                                                         --< 3
    greeting: string, bool -> string

func greeter(name: string, greet: bool, node: string, service_id: string) -> string :   --< 4
    on node:                                                      
      Greeting service_id
      res <- Greeting.greeting(name, greet)
    <- res
```

Reply:
do is still need Local for the TS client?

Reply:
No, in TS returning a value was supported by default